core/templates/dev/head/pages/splash-page/splash-page.controller.ts

```diff
@@ -18,19 +18,18 @@
 
 require('base-components/base-content.directive.ts');
 
-require('domain/utilities/UrlInterpolationService.ts');
 require('services/SiteAnalyticsService.ts');
 require('services/UserService.ts');
 
+const html = require('./splash-page.directive.html');
```

Comment:
Can we add this directly after `template:` instead of creating a constant, I think this would easier to replace later when we would wnat to use ngtools.

Reply:
Done